ink/yuki/punishment_edging

```diff
@@ -827,6 +851,36 @@ TODO set perform animations
 
 ->->
 
+TODO add call to tease_bank_joi in each edging segment and add more lines here
```

Comment:
I like where you're going with this. I want to make sure the scene doesn't feel like just another JOI scene, but without the ability to cum, so try to make sure the new lines aren't instructions. Maybe they can be comments on their performance or how much she's looking forward to playing with them as a good puppy.

Reply:
Edited, commented out more of the JOI style lines. :) Appreciate the feedback here.